CMakeLists.txt

```diff
@@ -297,48 +297,14 @@ toolchain_cc_warning_error_implicit_int()
 # 'cmake -DEXTRA_CFLAGS="-Werror -Wno-deprecated-declarations" ..'
 include(cmake/extra_flags.cmake)
 
-zephyr_cc_option(-fno-asynchronous-unwind-tables)
-zephyr_cc_option(-fno-pie)
-zephyr_cc_option(-fno-pic)
-zephyr_cc_option(-fno-strict-overflow)
-
-if(CONFIG_OVERRIDE_FRAME_POINTER_DEFAULT)
-  if(CONFIG_OMIT_FRAME_POINTER)
-    zephyr_cc_option(-fomit-frame-pointer)
-  else()
-    zephyr_cc_option(-fno-omit-frame-pointer)
-  endif()
-endif()
+toolchain_cc_others_1()
```

Comment:
"others" is a bit of a placeholder. How about: `toolchain_specific_options_group1()`?

Reply:
Yes, this entire patch is a placeholder, because the toolchain abstraction is incomplete, but I wanted compiler-specific options out of the top-level CMakeLists.txt because they're polluting my work on other, non-GCC, toolchains.  Those actually in charge of the toolchain abstraction will almostly certainly dissect/reorganize/refactor rename all of these things, so I don't want to get hung up on such minor details now.